mep/pages/templates/pages/home_page.html

```diff
@@ -1,5 +1,11 @@
 {% extends 'base.html' %}
 
+{% block metadata %}
+{# verification code for google search console #}
+<meta name="google-site-verification" content="pS2kpksxg6JeC90IvA8BHVsFuK_6b7J_vARVLqqu7ck" />
```

Comment:
what's this do? are we adding google search to the site?

Reply:
Verification for google webmaster tools. I had it in develop but hadn't pushed yet so it got pulled over when I updated from develop